### PR TITLE
v4.2.0

### PR DESCRIPTION
**Pull request by PHochmann**
This pull request merges the consolidated parts, picked file by file, of ci/december2021, extracted into branch ci/december2021-v4.2.0 into main. Changes (mainly small fixes of typos) are merged back into develop in a separate pull request. See https://github.com/International-Data-Spaces-Association/InformationModel/issues/550

Mainly, changes of ci/december2021 were breaking as they deleted ontology terms or changed them in an incompatible way.

### CODE REVIEW

**Comment by JohannesLipp** [comment]
@PHochmann you might want to add yourself to the author list :)

File: codes/Mediatype.ttl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix ids: <https://w3id.org/idsa/core/> .
+@prefix idsc: <https://w3id.org/idsa/code/> .
+
+# <https://www.iana.org/assignments/media-types/application/json>
```
Comment (JohannesLipp) [inline codes/Mediatype.ttl:6]: Remove the entire file as it is empty now?

File: examples/DATA3.ttl [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+@prefix conn3: <https://aastat.gov.de/connector/conn3/> .
+@prefix data3: <https://aastat.gov.de/connector/conn3/data3/> .
+@prefix part1: <https://im.internationaldataspaces.org/participant/part1> .
+
+data3:
+    a ids:TextResource ;
+    ids:title "European highway statistics - traffic report"@en ;
+    ids:description "Detailed traffic analysis report based on European highway statistics data 2000 - 2017."@en;
+    ids:keyword "report", "highway", "statistics", "Europe" ;
+    ids:temporalCoverage [
```
Comment (JohannesLipp) [inline examples/DATA3.ttl:17]: Replace with `dct:temporal`, `time:hasBeginning`, `time:Instant`, and `time:inXSDDateTimeStamp` like in https://github.com/International-Data-Spaces-Association/InformationModel/blob/7fd5177f7a7e21c8136af7cc8f22f1a2705ee967/examples/DATA2.ttl#L55 ?

File: version_mapping/3.0.0-3.1.0/UpgradeShapes-3.0.0-to-3.1.0.ttl [hunk 3]
```diff
@@ -130,15 +132,15 @@ ids:MessageUpgradeShape
 			$this ids:contractRejectionReason ?newContractRejectionReason .
 			idsm:VersionUpgrade ids:contractRejectionReason ?oldContractRejectionReason.
 			
-			$this ids:issued ?newIssued .
-			idsm:VersionUpgrade ids:issued ?oldIssued.
+			$this dct:issued ?newIssued .
```
Comment (JohannesLipp) [inline version_mapping/3.0.0-3.1.0/UpgradeShapes-3.0.0-to-3.1.0.ttl:135]: These things in the `version_mapping` directory must not be updated! They represent contents of older IM versions, and therefore must stay as they are. Please revert the "ids to dct/dcat" upgrades in both 3.0.0->3.1.0 and 3.1.0->4.0.0, thanks!

**Review by JohannesLipp — APPROVED** [review]
Good work! Please fix the 3 minor change requests, if possible. Thanks!

**Comment by JohannesLipp** [comment]
@clange please review, thanks!

**Review by clange — APPROVED** [review]
Looks great. Please
1. apply my suggestions, 
2. globally replace the mis-spelled `dct:licence` by `dct:license`

and then it will be ready for merge.

**Pull request merged**